bacs/kturle/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# Maintainer: Mikhail Ivchenko <ematirov@gmail.com>
+pkgname='kturtle.interpreter-git'
+pkgver=943.2fcaeef
+pkgrel=1
+pkgdesc="Kturtle customized for BACS"
+arch=('i686' 'x86_64')
+url="https://github.com/EgorMatirov/kturtle"
+license=('GPL' 'LGPL')
+groups=()
+depends=(
+  'qt5-base'
+)
+makedepends=(
+  'git'
+  'gcc'
+  'extra-cmake-modules'
+  'qt5-tools'
+)
+source=("$pkgname::git+https://github.com/EgorMatirov/kturtle.git")
+noextract=()
+md5sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/$pkgname"
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
```

Comment:
While it does not belong to this PR, please rename binary to `bacs.kturtle` or something with `bacs` prefix. https://github.com/EgorMatirov/kturtle/commit/24d4c2c0f0deb3fac93a2f00b348981b9fab34da is the rename commit but I would strongly prefer this fork to mention `bacs`.

Reply:
Done. https://github.com/EgorMatirov/kturtle/commit/301f66c5ecf18263f9a163c5f5798de733c6352a